app/views/facilities/_quicklinks.html.haml

```diff
@@ -0,0 +1,21 @@
+.quick_links 
+  .add_to_cart
+    - if not session_user
+      = link_to "#{t("admin.shared.add_to",model:t("pages.cart"))} (#{t("pages.login")})", new_user_session_path
+    - elsif session_user.can_override_restrictions?(product) || product.can_be_used_by?(acting_user)
+      - if not session_user.accounts.empty?
+        = link_to t("admin.shared.add_to",model:t("pages.cart")), add_order_path(acting_user.cart(session_user), order: {order_details: [{product_id: product.id, quantity: 1}] }), method: :put
+      - else
+        = t("facility_accounts.account_table.notice")
+    - else
+      = t("facility_account_orders.cannot_purchase")
```

Comment:
I would recommend putting the translation at `views.facilities.quicklinks` and then you should be able to access it with `text("quicklinks")` because the scope will be inferred from the partial's filename.

Reply:
What would you think of wrapping this in a in a `.label.label-important` class. (http://getbootstrap.com/2.3.2/components.html#labels-badges)